src/test/java/uk/gov/companieshouse/reconciliation/email/SendElasticsearchEmailRouteTest.java

```diff
@@ -72,11 +72,16 @@ public void configure() throws Exception {
                 .withHeader("ResourceLinkReference", "Compare Name Alpha Mongo Link")
                 .build();
 
+        Exchange fifthExchange = ExchangeBuilder.anExchange(context)
```

Comment:
Can we use more meaningful variable names here, please?

Reply:
Resolved in latest commit.